src/lib/features/NavMenu.svelte

```diff
@@ -45,7 +45,7 @@
   });
 </script>
 
-<nav class="gap-f32 text-primary font-title xs:hidden md:flex">
+<nav class="gap-f32 font-title xs:hidden text-black md:flex">
```

Comment:
We might need to add a dark mode option to the header, but that can be a separate issue.